qa-heal.planx-pla.net/manifests/hatchery/hatchery.json

```diff
@@ -99,7 +99,7 @@
       "cpu-limit": "1.0",
       "memory-limit": "2Gi",
       "name": "(Generic, Limited Gen3-licensed) Stata Notebook",
-      "image": "quay.io/cdis/jupyter-pystata-gen3-licensed:feat_stata-license",
+      "image": "quay.io/cdis/jupyter-pystata-gen3-licensed:master",
```

Comment:
since the PR has been merged 🎉 can we make a release of these 2 images and use the versioned images for deployment?